docs/source/configuration.rst

```diff
@@ -298,11 +298,10 @@ Prolific Recruitment
     key = value format of ``config.txt`` whenever possible, and leave ``prolific_recruitment_config``
     for complex requirements which can't be configured in this simpler way.
 
-``prolific_reward_cents`` *int*
-    Base compensation to pay your participants, in cents
+    .. note::
 
-    Prolific will use the currency of your researcher account, and convert automatically
-    to the participant's currency.
+        Prolific will use the currency of your researcher account, and convert automatically
```

Comment:
Why is this located here?

Reply:
I thought to leave it as a note.  Changed indentation, though.

Reply:
OK, I just think the location is now a bit weird, as it comes in the Prolific recruitment config section. Why not retain a heading for `prolific_reward_cents` that states that this config variable is no longer used, and people should now use the `base_payment` variable, but beware that they have to divide by 100?

Reply:
I have changed my mind about this, I don't really care.